tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -183,6 +185,11 @@ private SpanContext(TraceId traceId, string serviceName)
         /// </summary>
         internal TraceId TraceId128 { get; }
 
+        /// <summary>
+        /// Gets the 128-bit trace id.
+        /// </summary>
+        internal ulong TraceId64 { get; }
```

Comment:
Let's use the `TraceId128` (from above) and access the 64-bits from it.

Reply:
Ah wasn't sure if doing the conversion in the propagator class was more impactful, will remove this and access it there instead.